test/integration/apiserver/apply/apply_test.go

```diff
@@ -59,13 +60,13 @@ func setup(t testing.TB, groupVersions ...schema.GroupVersion) (*httptest.Server
 		controlPlaneConfig.ExtraConfig.APIResourceConfigSource = resourceConfig
 	}
 	controlPlaneConfig.GenericConfig.OpenAPIConfig = framework.DefaultOpenAPIConfig()
-	_, s, closeFn := framework.RunAnAPIServer(controlPlaneConfig)
+	_, _s, closeFn := framework.RunAnAPIServer(controlPlaneConfig)
```

Comment:
you should keep the `s` , it's being used lated for `s.URL`, the underscore is a typo , right?

Reply:
heh.. typo (yes, I know it's used here - it's not used only outside)